src/app/lib/colors.scss

```diff
@@ -17,7 +17,7 @@ $slate-gray-400: #708090;
 $white: #fff;
 $black: #000;
 $third-column-gray: #5e676f;
-$primary-blue: #00AEEF;
+$primary-blue: #00AEFF;
```

Comment:
This isn't really relevant to this pull requests. But what are your thoughts about the color names in this colors.scss? If there were ever some major change in or corporate color scheme `primary-blue` locks us into blue as our primary color. Would it make more sense for this name to be just `primary`?

Reply:
My suggestion @corykon would be to take the bull by the horns here and create a fork of Cashmere for yourself, then make a branch for updates to the colors.scss file.  And we can talk it through as a group if you have a draft for the OpenSpace meeting.  Or submit a pull request and we can talk it through there.